_includes/pillars.html

```diff
@@ -0,0 +1,32 @@
+<div class="row">
+    <div class="col-md-8 col-md-offset-2 text-center">
+      <p><b>MINISTRY OBJECTIVE</b></p>
+    </div>
+    <div class="col-md-8 col-md-offset-2 text-center">
+        <h2><b>Mature & Equipped Followers of Jesus</b></h2>
+    </div>
+    <div class="col-md-10 col-md-offset-1 text-center">
+        <p>
+            Christ Central’s whole ministry aims to make mature and equipped followers of Jesus Christ.  Simply put, we are in the business of making faithful disciples of Jesus.  That’s the goal, that’s the agenda for everyone who is part of Christ Central. What does a mature and equipped follower look like? There are four foundational pillars: Worship, Belong, Grow and Bless.
+        </p>
+    </div>
+</div>
+  
+<div class="row mb120">
+    <div class="col-sm-3 text-center">
```

Comment:
Aren't these columns too spaced apart covering the whole width of the section instead of being closer together in the center?

Reply:
made them `col-sm-2`s